<launch>

	<node pkg="person_detection" type="yolo_detector_seg.py" name="yolo_detector_seg" output="screen">
		<param name="weight_path"       value="$(find person_detection)/weights/yolov8n-seg_openvino_model" />
		<!-- <param name="weight_path"       value="$(find person_detection)/weights/yolov8n-seg.pt" /> -->

		<param name="info_topic"        value="/camera/depth/camera_info" />
		<param name="depth_topic"       value="/camera/depth/image_rect_raw" />
		<param name="image_topic"       value="/camera/color/image_raw" />
		<param name="conf"              value="0.6" />
		<param name="imgsz"             value="640" />
		<param name="use_cpu"           value="true" />
	</node>

	<node pkg="person_detection" type="marker.py" name="marker" output="screen" />

	<!-- <param name="robot_description" command="$(find xacro)/xacro -inorder '$(find realsense2_description)/urdf/test_d455_camera.urdf.xacro' use_nominal_extrinsics:=false" /> -->
	<!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find realsense2_description)/rviz/urdf.rviz" required="true" /> -->

</launch>